pshtt/utils.py

```diff
@@ -81,7 +81,7 @@ def format_domains(domains):
     formatted_domains = []
 
     for domain in domains:
-        # Replace a single instance of http://, and www. if present.
-        formatted_domains.append(domain.replace("http://", "", 1).replace("www.", "", 1))
+        # Replace a single instance of http://, https://, and www. if present.
+        formatted_domains.append(domain.replace("https://", "").replace("http://", "", 1).replace("www.", "", 1))
```

Comment:
If a domain is https://http://www.something.com this would result in something.com. I am not sure if this is okay or not, if not would need to add some logic in order to check for the removal. I could also change the line to use re.sub and a regular expression for replacing one instance of http?://. I thought a regex here might be a tad overkill though.